contracts/external/interestsetters/DoubleExponentInterestSetter.sol

```diff
@@ -119,14 +119,19 @@ contract DoubleExponentialInterestSetter is
             });
         }
 
-        uint256 result = 0;
-        uint256 polynomial = BASE;
+        // process the first coefficient
+        uint256 coefficients = s.coefficients;
+        uint256 result = uint8(coefficients) * BASE;
```

Comment:
what does `uint8(coefficients)` get you? the lowest or the highest bytes in the number? what does this represent?

Reply:
lowest byte. Same as doing (coefficients % 256). pretty sure solidity generates the same code too, just more readable this way (IMO)

Reply:
Actually I think `coefficients % 256` is more readable because it's not clear if `uint8(coefficients)` takes the highest or lowest byte. It's not a big deal though